codewof/programming/content/en/square-area/question.md

```diff
@@ -0,0 +1,4 @@
+# Square Area
+
+Write a function `square_area(side_length)` that **returns** the area of the square where the length of a side is `side_length`. 
+The formula for calculating the area of a square is a = s<sup>2</sup> where `a` is the area and `s` is the length of a side.
```

Comment:
Could you put the formula in backticks to be consistent?

Reply:
The only reason I didn't put it in as a code block was that superscripts cannot be used in code blocks - and I didn't want to literally give them the code. The options were:
- `a = s ** 2` which gives away the main point of the excercise
- `a = s ^ 2` which is confusing
- a = s<sup>2</sup> which is the notation they are used to but without the formatting placed on the other formulas
- (or `a = s`<sup>2</sup> but that is just ugly)

Do you think that `a = s ** 2` should be used?

Reply:
I see, okay in that case not having backticks is fine